.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Test
+on: [push, pull_request]
```

Comment:
Im Pull Request ist jetzt irgendwie alles doppelt gelaufen. Nicht sicher, wie man das richtig macht, vielleicht auch einfach egal.

Reply:
Es gibt da eine [GitHub-Action](https://github.com/fkirc/skip-duplicate-actions/) die Duplikate skippt. Die hab ich jetzt mal eingebaut, aber wie man unten in der Liste der Checks sehen kann, werden trotzdem auch die geskippten angezeigt. Immer noch nicht ganz ideal, aber etwas besser finde ich.

Reply:
Nehmen wir einfach mal so. Für externe Pull Requests aus einem anderen Repo werden ja dann wahrscheinlich nur die pull_request Checks ausgeführt.